packages/style-engine/src/styles/background/index.ts

```diff
@@ -43,15 +41,6 @@ const backgroundImage = {
 			} );
 		}
 
-		// If no background size is set, but an image is, default to cover.
-		if ( _backgroundSize === undefined ) {
-			styleRules.push( {
-				selector: options.selector,
-				key: 'backgroundSize',
-				value: 'cover',
-			} );
-		}
-
```

Comment:
The whole point of moving these out of the style engine was to ensure the style engine isn't making decisions for the consumer. It's job is (mainly) to parse values and return CSS and declarations.

Reply:
For existing block markup out in the wild, which might not have `backgroundSize` set, how will these styles be output in the editor?

Reply:
I think we can use `useBlockProps`, exported from the background hook and run through the `blocks.getSaveContent.extraProps` filter.